content/background.md

```diff
@@ -28,12 +28,13 @@ The central problem is that sharing a route via BGP relies on trust, in the sens
 
 Starting with Core 20.0, you can pass an ASmap file via the `-asmap=<filepath>` option. Sourcing the data and generating these ASmaps require some tooling.
 
-## Existing work
+## Prior work
 
-rrybarczyk wrote asmap-rs to download and parse RIS raw data from the RIPE NCC and produce a IP-to-AS map file.
+rrybarczyk wrote [asmap-rs](https://github.com/rrybarczyk/asmap-rs/) to download and parse RIS raw data from the RIPE NCC and produce a IP-to-AS map file.
 
-fjahr wrote kartograf sources data from RIPE, RPKI, and IRR databases, parses and combines the data, to produce an IP-to-AS map file.
+fjahr wrote [kartograf](https://github.com/fjahr/kartograf) sources data from RIPE, RPKI, and IRR databases, parses and combines the data, to produce an IP-to-AS map file.
```

Comment:
Hm, if this section is now prior work then I don't think kartograf belongs here because it's actively maintained.